tests/select_ai_feedback_test.py

```diff
@@ -0,0 +1,673 @@
+import logging
+import select_ai
+from select_ai.action import Action
+import pytest
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+PROFILE_NAME = "PYSAI_TEST_FEEDBACK_PROFILE"
```

Comment:
Use uuid just be safe because Github Actions runs tests concurrently and that can create concurrency issues accessing the profile. We run the testcases for multiple different Python versions in parallel.  

Make sure the profile name is unique. We can use test suite id e.g. 1500 and a unique identifier. 

```python
import uuid

PROFILE_NAME = f"PYSAI_1500_{uuid.uuid4().hex.upper()}"
```